Modern/utilities/src/test/java/org/bonej/utilities/ElementUtilTest.java

```diff
@@ -214,7 +214,7 @@ public void testIsColorsBinaryReturnsFalseIfIntervalEmpty() {
 	public void testIsColorsBinaryReturnsTrueForMonochrome() {
 		final IterableInterval<DoubleType> interval = ArrayImgs.doubles(2, 2);
 
-		final boolean result = ElementUtil.isColorsBinary(interval);
+		final boolean result = ElementUtil.isBinary(interval);
 
 		assertTrue("Monochrome image should be binary color", result);
 	}
```

Comment:
see above

Reply:
Good catch. I've removed references to colour from the tests.